compiler_opt/es/blackbox_evaluator.py

```diff
@@ -101,5 +101,62 @@ def get_results(
 
     return futures
 
-  def set_baseline(self) -> None:
+  def set_baseline(self, pool: FixedWorkerPool) -> None:
+    del pool  # Unused.
     pass
+
+
+@gin.configurable
+class TraceBlackboxEvaluator(BlackboxEvaluator):
+  """A blackbox evaluator that utilizes trace based cost modelling."""
+
+  def __init__(self, train_corpus: corpus.Corpus,
+               est_type: blackbox_optimizers.EstimatorType, bb_trace_path: str,
+               function_index_path: str):
+    self._train_corpus = train_corpus
+    self._est_type = est_type
+    self._bb_trace_path = bb_trace_path
+    self._function_index_path = function_index_path
+
+    self._has_baseline = False
+    self._baseline = 1
```

Comment:
what's the baseline, and why 1?

Reply:
Also, can't you model _baseline as `Optional[int]` or what, and then _has_baseline can go away because it's just `_baseline is not None`?